apps/sms/views/conversation/js/conversation.js

```diff
@@ -2262,16 +2258,23 @@ var ConversationView = {
 
   // FIXME/bug 983411: phoneNumber not needed.
   simSelectedCallback: function conv_simSelected(phoneNumber, cardIndex) {
-    if (Compose.isEmpty()) {
-      return;
-    }
-
     cardIndex = +cardIndex;
+
     if (isNaN(cardIndex)) {
       cardIndex = 0;
     }
 
-    this.sendMessage({ serviceId: cardIndex });
+    if(Compose.isEmpty()) {
+      return Utils.confirm(
```

Comment:
__nit:__ how about smth like this to have single entry point for ```sendMessage```?

```
    var sendConfirmationPromise = Compose.isEmpty() ?
      Utils.confirm(
        'sendEmptyMessage-confirmation', null, { className: 'danger' }
      ) :
      Promise.resolve();

    return sendConfirmationPromise.then(
      () => this.sendMessage({ serviceId: cardIndex })
    );
```

It will require minor changes in existing tests, but should not be too much